Interpreter: undefined reference to `llvm.bitreverse.i32' (Crystal::Loader::LoadError)

## Bug Report
Example code:
```crystal
require "bit_array"

ary = BitArray.new(64, false)
ary.reverse!

```
Interpreter gives below error:
```
undefined reference to `llvm.bitreverse.i32' (Crystal::Loader::LoadError)
  from src/compiler/crystal/loader.cr:78:27 in 'c_function'
  from src/compiler/crystal/interpreter/compiler.cr:1911:17 in 'visit'
  from src/compiler/crystal/syntax/visitor.cr:27:12 in 'accept'
  from src/compiler/crystal/semantic/bindings.cr:17:7 in 'compile_def'
  from src/compiler/crystal/interpreter/compiler.cr:2006:5 in 'create_compiled_def'
  from src/compiler/crystal/interpreter/compiler.cr:1805:22 in 'visit'
  from src/compiler/crystal/syntax/visitor.cr:27:12 in 'accept'
  from src/compiler/crystal/interpreter/compiler.cr:3094:5 in 'visit'
  from src/compiler/crystal/syntax/visitor.cr:27:12 in 'accept'
  from src/compiler/crystal/interpreter/compiler.cr:3094:5 in 'visit_primitive'
  from src/compiler/crystal/syntax/visitor.cr:27:12 in 'accept'
  from src/compiler/crystal/interpreter/compiler.cr:1244:8 in 'accept'
  from src/compiler/crystal/interpreter/compiler.cr:557:7 in 'accept'
  from src/compiler/crystal/semantic/bindings.cr:17:7 in 'compile_def'
  from src/compiler/crystal/interpreter/compiler.cr:2006:5 in 'create_compiled_def'
  from src/compiler/crystal/interpreter/compiler.cr:1805:22 in 'visit'
  from src/compiler/crystal/syntax/visitor.cr:27:12 in 'accept'
  from src/compiler/crystal/interpreter/compiler.cr:3094:5 in 'compile_call_arg'
  from src/compiler/crystal/interpreter/compiler.cr:2204:7 in 'visit'
  from src/compiler/crystal/syntax/visitor.cr:27:12 in 'accept'
  from src/compiler/crystal/interpreter/compiler.cr:557:7 in 'accept'
  from src/compiler/crystal/semantic/bindings.cr:17:7 in 'compile_def:closure_owner'
  from src/compiler/crystal/interpreter/compiler.cr:3045:5 in 'accept'
  from src/compiler/crystal/interpreter/compiler.cr:557:7 in 'accept'
  from src/compiler/crystal/interpreter/compiler.cr:3286:11 in 'compile'
  from src/compiler/crystal/interpreter/interpreter.cr:215:5 in 'interpret'
  from src/compiler/crystal/interpreter/repl.cr:146:5 in 'interpret_and_exit_on_error'
  from src/gc/boehm.cr:129:5 in 'repl'
  from src/compiler/crystal/command.cr:100:7 in '__crystal_main'
  from src/crystal/main.cr:115:5 in 'main'
  from /lib/x86_64-linux-gnu/libc.so.6 in '??'
  from /lib/x86_64-linux-gnu/libc.so.6 in '__libc_start_main'
  from /home/chao/git/personal/crystal/.build/crystal in '_start'
  from ???
```

--- a/src/compiler/crystal/interpreter/instructions.cr
+++ b/src/compiler/crystal/interpreter/instructions.cr
@@ -1854,6 +1854,15 @@ require "./repl"
           end,
         },
       {% end %}
+
+      {% for n in [16, 32, 64] %}
+        interpreter_intrinsics_bitreverse{{n}}: {
+          pop_values: [value : UInt{{n}}],
+          push:       true,
+          code:       LibIntrinsics.bitreverse{{n}}(value),
+        },
+      {% end %}
+
       libm_ceil_f32: {
         pop_values: [value : Float32],
         push:       true,

--- a/src/compiler/crystal/interpreter/primitives.cr
+++ b/src/compiler/crystal/interpreter/primitives.cr
@@ -452,6 +452,15 @@ class Crystal::Repl::Compiler
     when "interpreter_intrinsics_counttrailing128"
       accept_call_args(node)
       interpreter_intrinsics_counttrailing128(node: node)
+    when "interpreter_intrinsics_bitreverse16"
+      accept_call_args(node)
+      interpreter_intrinsics_bitreverse16(node: node)
+    when "interpreter_intrinsics_bitreverse32"
+      accept_call_args(node)
+      interpreter_intrinsics_bitreverse32(node: node)
+    when "interpreter_intrinsics_bitreverse64"
+      accept_call_args(node)
+      interpreter_intrinsics_bitreverse64(node: node)
     when "interpreter_libm_ceil_f32"
       accept_call_args(node)
       libm_ceil_f32 node: node

--- a/src/intrinsics.cr
+++ b/src/intrinsics.cr
@@ -49,8 +49,13 @@ lib LibIntrinsics
   {% if flag?(:interpreted) %} @[Primitive(:interpreter_intrinsics_read_cycle_counter)] {% end %}
   fun read_cycle_counter = "llvm.readcyclecounter" : UInt64
 
+  {% if flag?(:interpreted) %} @[Primitive(:interpreter_intrinsics_bitreverse64)] {% end %}
   fun bitreverse64 = "llvm.bitreverse.i64"(id : UInt64) : UInt64
+
+  {% if flag?(:interpreted) %} @[Primitive(:interpreter_intrinsics_bitreverse32)] {% end %}
   fun bitreverse32 = "llvm.bitreverse.i32"(id : UInt32) : UInt32
+
+  {% if flag?(:interpreted) %} @[Primitive(:interpreter_intrinsics_bitreverse16)] {% end %}
   fun bitreverse16 = "llvm.bitreverse.i16"(id : UInt16) : UInt16
 
   {% if flag?(:interpreted) %} @[Primitive(:interpreter_intrinsics_bswap32)] {% end %}

